source/agora/node/Config.d

```diff
@@ -224,6 +224,10 @@ public struct NodeConfig
     /// from all other nodes
     public Duration block_catchup_interval = 20.seconds;
 
+    /// The number of the other nodes from which a node can retrive missing
```

Comment:
I think you need to update this comment.

Reply:
Oh, I missed it. Will update.

Reply:
Updated.